src/util.c

```diff
@@ -834,7 +834,7 @@ int write_result_into_log_file(struct ntttcp_test_endpoint *tep)
 	fprintf(logfile, "		<no_stdio_buffer>%s</no_stdio_buffer>\n", "False");
 	fprintf(logfile, "		<throughput_Bpms>%d</throughput_Bpms>\n", 0);
 	fprintf(logfile, "		<cpu_burn>%d</cpu_burn>\n", 0);
-	fprintf(logfile, "		<latency_measurement>%s</latency_measurement>\n", tep->endpoint_role == ROLE_SENDER ? "True" : "False");
+	fprintf(logfile, "		<latency_measurement>%s</latency_measurement>\n", "False");
```

Comment:
Write "False" in all cases? How about if your RTT latency fetched successfully?

Reply:
I didn't set it to "False" for all cases. It was `	fprintf(logfile, "		<latency_measurement>%s</latency_measurement>\n", "False");` in the original code. I think github's code review is based on the diff between the last commit and the second last one. I changed it to `fprintf(logfile, "		<latency_measurement>%s</latency_measurement>\n", tep->endpoint_role == ROLE_SENDER ? "True" : "False");` but I reverted this change later on because `latency_measurement` in Windows NTTTCP is a metric from App level. I don't want to make it inconsistent.

Reply:
`latency_measurement` is currently not implemented.